Update sidebar.markdown

benblb opened the pull request:
Added info on where to find the option to enable sidebar. Really struggled to enable it and wanted to contribute to hopefully help someone else.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

benblb committed:
Update sidebar.markdown

Added info on where to find the option to enable sidebar. Really struggled to enable it and wanted to contribute to hopefully help someone else.

homeassistant commented:
Hi @benblb, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19132).

Once you do that we will be able to review and accept this pull request.

Thanks!

ludeeus commented on source/_lovelace/sidebar.markdown:
```diff
@@ -10,7 +10,7 @@ The sidebar view has 2 columns, a wide one and a smaller one on the right.
 
 To change a view to edit mode, or to change the location of a card, enable edit mode:
-Click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**.
+Click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Then, select your [view (Lovelace tab)](https://www.home-assistant.io/lovelace/views/) and click the edit icon next to its name. At the bottom of the view configuation screen, select the "View type" called sidebar.  
```
This should not be a part of this page, it should be in <https://www.home-assistant.io/lovelace/views/> that would cover all 3 types and how to change between them.

This page can link to that, but the documentation for that belongs under <https://www.home-assistant.io/lovelace/views/>

benblb replied:
I see what you mean @ludeeus, but that's pretty confusing for the user to dig through multiple pages when all they want to know is how to enable sidebar. It's odd that the sidebar page doesn't explain how to enable the feature.

ludeeus replied:
I think this page should have it but in form of a link that goes to the main view page.
Changing the mode is a part of the view configuration, it would be more confusing if it's not a part of that.

benblb replied:
I think that forces the user into an unnecessary click. What if we explain how to enable and add "for more info, see the views page" and link there? 

Sidebar is a new feature and I foresee many people being confused and simply looking for how to enable. The views page is pretty busy and would require the user to look through quite a bit.

bramkragten replied:
There is a difference between changing an existing view to a sidebar view, or creating a new one. The creating should be on this page, changing an existing view should be on <home-assistant.io/lovelace/views> IMHO

bramkragten replied:
But it should def not be in the part that is about editing cards in your sidebar view, as at that point you should already have a sidebar view.

benblb replied:
That makes sense to me. What about...

```suggestion
Click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Create a new [view (Lovelace tab)](https://www.home-assistant.io/lovelace/views/) and at the bottom of the view configuration screen select the view type called sidebar.  
```

benblb committed:
Update source/_lovelace/sidebar.markdown

Instruction for creating a new sidebar view, not editing a current view to become sidebar.

balloob commented:
I don't think that this change makes sense. There are 5 inputs on that form, one of them is "View Type", no need to specify "at the bottom". 

It's also added to a sentence that talks about "editing", not creating new.
